README.rst

```diff
@@ -53,29 +56,42 @@ http://nitrate.readthedocs.org/
 How to run
 ----------
 
-There are a series of documentation for installing Nitrate in
-`Installation Guide`_. Choose one for your case. It is appreciated to catch any
-issues of the documentation, report it or fix it.
+There are a series of instructions for installing Nitrate in the
+`Installation Guide`_. Skim through the documentation and choose the
+installation instructions that are appropriate for your case. And please, if
+you identify any issues with the installation guide, kindly bring it to our
+attention. You can either report the issue on the github repo, or submit a PR
+with a fix for it.
 
 .. _Installation Guide: https://nitrate.readthedocs.io/en/latest/install/index.html
 
 Contribution
 ------------
 
-Any kind of contribution is welcome, whatever to the documentation, code or
-even just ideas to make Nitrate better. Please refer to Contribution_ for more
+Any kind of contribution is highly welcome and welcome, whether to the
+documentation or the source code itself. We also greatly appreciate
+contributions in the form of ideas to make Nitrate better.
+
+Please refer to Contribution_ for more
 information on how to contribute.
 
-If you would like to write some code, the Vagrant machine would be a good
-choice for you to setup a development environment quickly, where you can run
-tests and debug issues. Refer to :ref:`The Vagrant Way <how_to_run>`.
+Contributing Code
+-----------------
+
+If you would like to write some code, the Vagrant machine would be a
+good choice for you to setup a development environment quickly, where you
+can run tests and debug issues.
+Please refer to the `Installation Guide`_ for more information on
```

Comment:
Are these two paragraphs expected in the rendered result? If yes, a blank line is required.

Reply:
Oh, no, you're expressing yourself accurately.

It's just my lack of understanding of RST showing up :man_facepalming:

So, I'll update my knowledge on RST and try again.